DailyReportWeb/Helpers/PolicyService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using SourceControlLogReporter.Model;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Web;
+
+namespace DailyReportWeb.Helpers
+{
+    public class PolicyService
+    {
+        public static void SetPolicyFullDraft(Policy policy, string policyPath)
+        {
+            policy.AdvancedOptions.NoIndividualDraft = true;
+            policy.AdvancedOptions.NoDraft = false;
+            policy.GeneratedProperties.IndividualDrafts = null;
+            policy.SaveToFile(policyPath);
+        }
+
+        public static void SetPolicyFinalReport(Policy policy, string policyPath)
+        {
+            policy.AdvancedOptions.NoDraft = true;
```

Comment:
Pai mai dar astea sunt porprietati care tin de setarile tale pentru acel policy. Se vor putea modifica din editor. 

Ele spun daca vrei sa primesti draft individual respectiv daca vrei sa primesti draft final. Nu spun daca s-a trimis in ziua respectiva sau nu. 

Tu banuiesc ca la un moment dat le resetezi, dupa ce se trimite si raportul final sa zicem. Dar cum stii care erau valorile intiale, daca tu ai scris peste ele?